docs/reference/content/driver/tutorials/ssl.md

```diff
@@ -155,10 +155,12 @@ MongoClientOptions.builder()
         .build();
 ```
 
-## JVM System Properties for TLS/SSL
+## Certificates
```

Comment:
This section is relevant only in situations when users want to use the JVM default truststore and keystore, instead of using separate ones when communicating with a MongoDB server (separate truststore/keystore can be specified via [`javax.net.ssl.SSLContext.init(KeyManager[] km, TrustManager[] tm, SecureRandom random)`](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLContext.html#init-javax.net.ssl.KeyManager:A-javax.net.ssl.TrustManager:A-java.security.SecureRandom-)). I believe it would be useful to reflect this info in the section title and the section content. For example:

### Configuring the default truststore and keystore (as an alternative to "Certificates")
One may either configure separate truststores and keystores via [`javax.net.ssl.SSLContext.init(KeyManager[] km, TrustManager[] tm, SecureRandom random)`](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLContext.html#init-javax.net.ssl.KeyManager:A-javax.net.ssl.TrustManager:A-java.security.SecureRandom-), or use the JVM default ones. 
#### Default truststore (as an alternative to "Certificate authority")
...content suggested in this PR
#### Default keystore (as an alternative to "Client certificates")
...content suggested in this PR